Tests/integration/Users.Tests.ps1

```diff
@@ -0,0 +1,113 @@
+#
+# Copyright 2022, Alexis La Goutte <alexis dot lagoutte at gmail dot com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+. ../common.ps1
+
+
+BeforeAll {
+    Connect-ArubaCX @invokeParams
+}
+
+Describe "Get Users" {
+    It "Get Users Does Not Throw an error" {
+        {
+            Get-ArubaCXUsers
+        } | Should -Not -Throw
+    }
+
+    #It "Get Users (admin) and confirm (via Confirm-ArubaCXUsers)" {
```

Comment:
Why leave it commented ? Maybe use <# #> ?

Reply:
for futur use when add support of Add/Set/Remove Users...